mcp/index.php

```diff
@@ -713,33 +10,24 @@ private function JSONerrorResponse($message, $code = -32603, $id = null, $contex
 		$server->handleHTTPRequest();
 	} elseif ($server->canBeEnabled) {
 		// if the MCP server passed the canBeEnabled check, but is not enabled, return a 200 OK response with a JSON payload indicating that the server is not enabled
-		header('Content-Type: application/json; charset=utf-8');
-		header('Cache-Control: no-cache, no-store, must-revalidate');
-		http_response_code(200);
-		echo json_encode([
+		$content = [
 			'status' => 'canBeEnabled',
 			'message' => 'MCP Server can be enabled',
 			'code' => 200,
 			'timestamp' => date('Y-m-d H:i:s')
-		]);
+		];
+		$server->sendResponse($content);
 	} else {
 		// If the MCP server is disabled, return a 503 Service Unavailable response
-		header('Content-Type: application/json; charset=utf-8');
-		header('Cache-Control: no-cache, no-store, must-revalidate');
-		http_response_code(503);
-		echo json_encode([
-			'error' => 'MCP Server is disabled',
-			'message' => 'Please enable the MCP Server in the Formulize settings.',
-			'code' => 503,
-			'timestamp' => date('Y-m-d H:i:s')
-		]);
-	}
-} catch (Exception $e) {
-	header('Content-Type: application/json; charset=utf-8');
-	header('Cache-Control: no-cache, no-store, must-revalidate');
-	http_response_code(500);
-	echo json_encode([
-		'error' => 'Server initialization failed: ' . $e->getMessage(),
-		'timestamp' => date('Y-m-d H:i:s')
-	]);
+		throw new FormulizeMCPException('MCP Server is disabled', 'server_disabled');
+	}
+} catch (FormulizeMCPException $e) {
+	$server->sendResponse([
+		'jsonrpc' => '2.0',
```

Comment:
This is perhaps where my other comment about the jsonrpc and result and id structure, runs into trouble... because not all responses have that structure. :(

Still, if there's a way to steamline what the lower methods are returning, I think that's good?